DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests/BasicAuthenticatedAccessibilityTests.cs

```diff
@@ -75,6 +75,9 @@ public class BasicAuthenticatedAccessibilityTests : AccessibilityTestsBase,
             "/TrackingSystem/Delegates/CourseDelegates/DelegateProgress/243104/EditSupervisor",
             "Edit supervisor for Digital Literacy for the Workplace - CC Test"
         )]
+        [InlineData("/TrackingSystem/Delegates/CourseDelegates/DelegateProgress/243104/EditCompletionDate",
+            "Edit completion date for Digital Literacy in the Workplace - CC Test"
+        )]

```

Comment:
This test has an error currently on jenkins, looks like the course name has changed somehow recently (might be worth asking on slack, I assume this passes for you locally):
   Expected h1Element.Text to be equivalent to 
"Edit completion date for Digital Literacy in the Workplace - CC Test" with a length of 68, but 
"Edit completion date for Digital Literacy for the Workplace - CC Test" has a length of 69, differs near "for" (index 42).

Reply:
oh no that was failing locally I didn't notice, I've updated it now but the build is still failing it looks like a same time test is not working for a few builds today

Reply:
OK that's fine then. Wasn't sure if it was a bad unit test updating the data, or someone that forgot to say they've done so. Yeah, ignore the final failing test, there is already a PR to fix it.